src/client/utils/funcs/fieldValueToString.ts

```diff
@@ -1,17 +1,35 @@
-import type { Field } from "../../../shared/manifest";
+import type { Field, ManifestRelation } from "../../../shared/manifest";
 
-export const fieldValueToString = (field: Field, value: any) => {
-  if (field.type === "date") {
-    value = `${value.day}/${value.month}/${value.year}`;
+export const fieldValueToString = (
+  field: Field | ManifestRelation,
+  value: any
+) => {
+  if (!field || !value) return value;
+  switch (field.type) {
+    case "relatesToOne":
+      value = value;
+      break;
+    case "relatesToMany":
+    case "ownsMany":
+      value = value;
+      break;
+    case "ownsOne":
+      value = value;
+      break;
+    case "date":
+      value = `${value.day}/${value.month}/${value.year}`;
+      break;
+    case "select":
+      value = field.options.find((o) => o.value === value)?.value;
+      break;
+    case "multiselect":
+      value = value
+        .map((v: string) => field.options.find((o) => o.value === v)?.value)
+        .join(", ");
```

Comment:
I advice separating with pipes instead of commas in case the options in the select have commas inside

Reply:
This is a good idea but it requires a couple of edits in other files and there's hurry with this PR. I'll take note and improve it another time.